options/locale/locale_en-US.ini

```diff
@@ -250,58 +250,58 @@ TreeName = File path
 Content = Content
 
 require_error = ` cannot be empty.`
-alpha_dash_error = ` must be valid alphanumeric or dash(-_) characters.`
-alpha_dash_dot_error = ` must be valid alphanumeric, dash(-_) or dot characters.`
-git_ref_name_error = ` must be a well formed git reference name.`
+alpha_dash_error = ` should contain only alphanumeric, dash ('-') and underscore ('_') characters.`
```

Comment:
this one should have `or` like one below or next one should be changed to `and` so that both have same

Reply:
Good catch! I changed the second one into `and`, since it's an inclusive enumeration.